contracts/leaser/src/tests/contract_tests.rs

```diff
@@ -33,20 +36,26 @@ type TheCurrency = Usdc;
 
 const DENOM: &str = TheCurrency::TICKER;
 const MARGIN_INTEREST_RATE: Percent = Percent::from_permille(30);
+const MIN_ASSET: Coin<TheCurrency> = Coin::<TheCurrency>::new(MIN_ASSET_AMOUNT);
+const MIN_SELL_ASSET: Coin<TheCurrency> = Coin::<TheCurrency>::new(MIN_SELL_ASSET_AMOUNT);
```

Comment:
not necessary to pass the type parameter on calling `new`, it's deduced from the const type